haystack_experimental/components/extractors/llm_metadata_extractor.py

```diff
@@ -240,28 +245,73 @@ def from_dict(cls, data: Dict[str, Any]) -> "LLMMetadataExtractor":
             deserialize_secrets_inplace(data["init_parameters"]["generator_api_params"], keys=["api_key"])
         return default_from_dict(cls, data)
 
+    def _extract_metadata_and_update_doc(self, document, errors, content):
+        """
+        Extract metadata from the content and update the document with the extracted metadata. If the extraction fails,
+        the error message will be stored in `llm_errors`.
+
+        :param document: Document to be updated with the extracted metadata.
+        :param errors: Dictionary to store error messages if the extraction fails.
+        :param content: Content to extract metadata from.
+        """
+        prompt_with_doc = self.builder.run(
+            template=self.prompt,
+            template_variables={self.prompt_variable:content}
+        )
+        print(f"Prompt with doc: {prompt_with_doc}")
```

Comment:
Let's remove the print statement or if we want to keep let's make it a logging statement.